src/sdk/strategy.ts

```diff
@@ -0,0 +1,121 @@
+import { SecretKey } from '@nucypher/nucypher-core';
+import { ethers } from 'ethers';
+
+import { EnactedPolicy } from '../policies/policy';
+import { Alice } from '../characters/alice';
+import { Bob } from '../characters/bob';
+import { Enrico } from '../characters/enrico';
+import { tDecDecrypter } from '../characters/universal-bob';
+
+import { Cohort } from './cohort';
+
+export class Strategy {
+  public policy?: EnactedPolicy;
+  public encrypter?: Enrico;
+  public decrypter?: tDecDecrypter;
+  constructor(
+    public cohort: Cohort,
+    public startDate: Date,
+    public endDate: Date,
+    private alice: Alice,
+    private bob: Bob,
+    private bobSecretKey: SecretKey,
+    public deployed: boolean
+  ) {}
+
+  public static create(
+    cohort: Cohort,
+    startDate: Date,
+    endDate: Date,
+    porterUri: string,
+    provider: ethers.providers.Web3Provider,
+    aliceSecretKey?: SecretKey,
+    dkgAlice?: boolean
+  ) {
+    if (dkgAlice == true) {
+      throw new TypeError('DKG Alice is not yet implemented');
+    }
```

Comment:
If so, maybe we could simplify the method signature here? And avoid some confusion as to the availability of this feature.
```suggestion
  ) {
```